exporters/geneva-trace/include/opentelemetry/exporters/geneva/geneva_fields.h

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021 (c) Microsoft Corporation.  All rights reserved.
+//
+#pragma once
+
+/* clang-format off */
+
+/**
+ * List of configurable Field Name constants:
+ *
+ * - env_ver                      - Schema version (optional for ETW exporter).
+ * - env_name                     - Built-in ETW name at envelope level (dedicated ETW field).
+ * - env_time                     - Built-in ETW time at envelope level.
+ * - env_dt_spanId                - OT SpanId
+ * - env_dt_traceId               - OT TraceId
+ * - startTime                    - OT Span start time
+ * - kind                         - OT Span kind
+ * - name                         - OT Span name in ETW 'Payload["name"]'
+ * - parentId                     - OT Span parentId
+ * - links                        - OT Span links array
+ *
+ * Other standard fields (reserved names) that may be appended by ETW channel:
+ *
+ * - Level
+ * - ProviderGuid
+ * - ProviderName
+ * - OpcodeName
+ * - KeywordName
+ * - TaskName
+ * - ChannelName
+ * - EventMessage
+ * - ActivityId
+ * - Pid
+ * - Tid
+ *
+ */
+#  define ETW_FIELD_VERSION           "env_ver"               /* Event version           */
+#  define ETW_FIELD_TYPE              "env_type"              /* Event type              */
+#  define ETW_FIELD_NAME              "env_name"              /* Event name              */
+
+#ifndef HAVE_FIELD_TIME
+#define HAVE_FIELD_TIME
+#endif
+#  define ETW_FIELD_TIME              "env_time"              /* Event time at envelope  */
```

Comment:
Should this `ETW_FIELD_TIME` be included in the above `#ifndef HAVE_FIELD_TIME / #endif`?